plugin/fanout/utils.go

```diff
@@ -0,0 +1,60 @@
+package fanout
+
+import (
+	"context"
+	"runtime/debug"
+	"time"
+
+	"github.com/coredns/coredns/plugin/dnstap"
+	"github.com/coredns/coredns/plugin/dnstap/msg"
+	"github.com/coredns/coredns/request"
+
+	tap "github.com/dnstap/golang-dnstap"
+	"github.com/miekg/dns"
+)
+
+func logIfNotNil(err error) {
+	if err == nil {
+		return
+	}
+	stack := debug.Stack()
+	log.Errorf("err :%v \nstack:\n%v", err, string(stack))
+}
+
+func toDnstap(ctx context.Context, host string, protocol string, state request.Request, reply *dns.Msg, start time.Time) error {
```

Comment:
@miekg I copied this func from forward because at forward plugin https://github.com/coredns/coredns/blob/master/plugin/forward/dnstap.go#L15 it is private. Can I make this func at forward plugin as public and reuse it for this plugin?